src/crystal/system/unix/file.cr

```diff
@@ -133,7 +133,7 @@ module Crystal::System::File
   def self.rename(old_filename, new_filename)
     code = LibC.rename(old_filename.check_no_null_byte, new_filename.check_no_null_byte)
     if code != 0
-      raise ::File::Error.from_errno("Error renaming file", file: old_filename, other: new_filename)
+      return ::File::Error.from_errno("Error renaming file", file: old_filename, other: new_filename)
```

Comment:
Redundant return

```suggestion
      ::File::Error.from_errno("Error renaming file", file: old_filename, other: new_filename)
```

Reply:
:thinking: You're replacing `raise` with `return`.  I don't think that's desired. Ditto below.

Reply:
:see_no_evil: sorry NVM, I misread the git diff.

Reply:
@caspiano its not redundant, method returns optional Error.

Reply:
Since it’s the last expression, the `return` can be dropped. They’re equivalent

Reply:
aah, i was confusing this PR with sth else :smile: . Thanks @caspiano